index.bs

```diff
@@ -4777,6 +4777,92 @@ which should make use of the existing browser permissions framework for the Geol
 
 The privacy principles in [[!FIDO-Privacy-Principles]] also apply to this specification.
 
+
+## De-anonymization prevention measures ## {#sctn-privacy-attacks}
+
+[INFORMATIVE]
+
+Many aspects of the design of the [=Web Authentication API=] are motivated by privacy concerns. The main concern considered in
+this specification is the protection of the user's personal identity, i.e., the identification of a human being or a correlation
```

Comment:
Shall I also add that as a non-normative definition reference?

Reply:
sure :)

Reply:
Done; I'm not sure it belongs in the bibliography, though, so I didn't add it to there.

Reply:
1) I'm nervous about using legal terms unless a lawyer has reviewed them.
2) "Historically, a human being was not necessarily a natural person in some jurisdictions where slavery existed (subject of a property right) rather than a person." makes me think we'd prefer "human being" here anyway.

Reply:
Valid points. @equalsJeffH What do you think, do we revert this?

Reply:
oh geeze....  I'd still use natural person but just not link it to wikipedia, or sure, use human being....

Reply:
after grovelling in the spec on another PR, i note that we use "human" in other terms/phrases, such as [human-palatable](https://w3c.github.io/webauthn/#human-palatability), so I'm thinking "human being" is fine. silly me.